rspace/src/test/scala/coop/rchain/rspace/StorageExamplesTests.scala

```diff
@@ -281,6 +281,19 @@ class InMemoryStoreStorageExamplesTests extends StorageExamplesTests {
   }
 }
 
+class ImmutableInMemoryStoreStorageExamplesTests extends StorageExamplesTests {
```

Comment:
It would make sense to also test this implementation using `IStoreTests`. Not necessarily by extending here. See `InMemoryStoreTests`